cob_description/urdf/sensors/rplidar.urdf.xacro

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <xacro:include filename="$(find cob_description)/urdf/sensors/sick_s300_laser.gazebo.xacro" />
+
+  <xacro:property name="cylinder_radius" value="0.08" />  <!--simple collision model for the laser -->
+  <xacro:property name="cylinder_length" value="0.05" />
+
+  <xacro:macro name="rplidar" params="name parent *origin">
+    <joint name="${name}_joint" type="fixed">
+      <xacro:insert_block name="origin" />
+      <parent link="${parent}_link"/>
+      <child link="${name}_link"/>
+    </joint>
+
+    <link name="${name}_link" type="laser">
+      <xacro:cylinder_inertial radius="${cylinder_radius}" length="${cylinder_length}" mass="0.5">
+        <origin xyz="0 0 0" rpy="0 0 0" />
+      </xacro:cylinder_inertial>
+      <visual>
+        <origin xyz="0 0 0" rpy="0 0 0" />
+        <geometry>
+          <mesh filename="package://cob_description/meshes/sensors/sick_s300_laser.dae"/>
+          <!--box size="0.001 0.001 0.001" /-->
+        </geometry>
+        <material name="IPA/LightGrey" />
+      </visual>
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0" />
+        <geometry>
+          <mesh filename="package://cob_description/meshes/sensors/sick_s300_laser_collision.stl"/>
+        </geometry>
+      </collision>
+    </link>
+
+    <!-- gazebo extensions -->
+    <xacro:sick_s300_laser_gazebo_v0 name="${name}"/>
```

Comment:
I'd prefer that this sensor gets it own gazebo.xacro!
(I assume specs are not identical to sick s300...)